pmm/pmm2-upgrade-portal-tests.groovy

```diff
@@ -127,6 +152,11 @@ pipeline {
                     which chromium-browser
                     sudo yum -y install mysql
                     sudo ln -s /usr/bin/chromium-browser /usr/bin/chromium
+                    sudo mkdir -p /srv/pmm-qa || :
+                    pushd /srv/pmm-qa
+                        sudo git clone --single-branch --branch \${PMM_QA_GIT_BRANCH} https://github.com/percona/pmm-qa.git .
+                        sudo git checkout \${PMM_QA_GIT_COMMIT_HASH}
```

Comment:
1 it will not work as you need to enter directory where you checked out git repository
```
corvin@amber:~/test_pmm$ git clone --single-branch --branch pmm-2.32.0 https://github.com/percona/pmm-qa.git
Cloning into 'pmm-qa'...
remote: Enumerating objects: 8474, done.
remote: Counting objects: 100% (988/988), done.
remote: Compressing objects: 100% (571/571), done.
remote: Total 8474 (delta 691), reused 648 (delta 413), pack-reused 7486
Receiving objects: 100% (8474/8474), 345.89 MiB | 5.42 MiB/s, done.
Resolving deltas: 100% (4012/4012), done.
corvin@amber:~/test_pmm$ git checkout adf5b2497f3eb4b8bd13374fd9d1e5058726f1ad
fatal: not a git repository (or any of the parent directories): .git
```
2 I wasn't able to find where you defined PMM_QA_GIT_BRANCH variable

Reply:
1. we have a dot in at the end of the command so the pmm-qa will be pulled into a folder in which we are in `/srv/pmm-qa`
2. PMM_QA_GIT_BRANCH is set as parameter when starting job.
```
        string(
            defaultValue: 'main',
            description: 'Tag/Branch for pmm-qa repository',
            name: 'PMM_QA_GIT_BRANCH')
```